patterns/1-initial/public-administration-and-suppliers.md

```diff
@@ -0,0 +1,78 @@
+## Title
+
+InnerSource and Collaboration in Public Administrations
```

Comment:
Taking a step back: You submitting a pattern about InnerSource in public admin to me means InnerSource adoption in the public sector to the degree where there's enough experience to talk about it (semi)publicly - YEAH! That is awesome to read here! Especially with the challenges in the public compared to the private sector.